app/controllers/concerns/part_of_claim_journey.rb

```diff
@@ -25,25 +25,24 @@ def send_unstarted_claimants_to_the_start
     redirect_to current_policy.start_page_url unless current_claim.persisted?
   end
 
-  # Keep this for StudentLoans and MathsAndPhysics journeys
-  # Keep ECP journey working for now with current_claim as we build out combined journey
   def current_claim
     @current_claim ||= claim_from_session || build_new_claim
   end
 
-  # TASK: does NOT support multiple claims yet!
   def claim_from_session
-    Claim.find(session[:claim_id]) if session.key?(:claim_id)
+    if session.key?(:claim_id)
+      claim_ids = session[:claim_id].split(",")
```

Comment:
Can't we have an array in the sesssion? It should just work if you don't use join/split by `,` and have claim_ids as `claims.pluck(:id)`.

Reply:
Actually, maybe not. Because then you'd have to handle an Array everywhere. Your take has the same results for one claim in the session.

Reply:
I think it works my new PR has it in, good feedback though!